examples/security/webserver-digest-auth/src/test/java/io/helidon/security/examples/webserver/digest/WebClientAuthenticationService.java

```diff
@@ -56,7 +56,7 @@ public WebClientServiceResponse handle(Chain chain, WebClientServiceRequest requ
         if (challenge == null) {
             return response;
         }
-        String uri = request.uri().path();
+        String uri = request.uri().path().path();
```

Comment:
`.path().path()` looks a bit odd. Is this to convert to String?

Reply:
yes, there are two methods on `UriPath` - `path()` and `rawPath()` to keep it consistent with all other URI components. I do agree it looks odd, we may want to do some renaming of methods here